packages/orchestrator-ui-components/src/components/WFOSubscription/WFOSubscription.tsx

```diff
@@ -59,17 +57,25 @@ export const Subscription: FC<SubscriptionProps> = ({ subscriptionId }) => {
     //         ).node,
     // };
 
+    const graphQLClient = new GraphQLClient(graphqlEndpointCore);
+
     const fetchSubscriptionOutline = async () => {
         console.log('Fetch outline query results for ID: ', subscriptionId);
```

Comment:
console.log is a bit evil, isn't it.. I vaguely remember from the past that I used some library for that so that you can easily toggle this.

Reply:
Yes indeed it doesn't belong there. For now I kept de scope to only move to lib. A different story is finalizing the component itself.